LG-4325 confirm the data formats for all IAL2 attributes (#170)
18F/identity-dev-docs: _pages/attributes.md

Here are the possible attributes that can be requested at a given IAL. This table contains the available user attributes, the IAL they are associated with, and how they can be accessed in OpenID Connect and SAML.

It is important to expect any number of characters in the `(string)` datatype unless directly followed by a number such as `(string36)`.

The following fields are stored together in a text datatype with a maximum length of 65,535 bytes:<br>

> First name, Last name, Address, Phone, Date of birth, Social security number

Reviewer: Is this useful to people? That's also the length of the blob before it's encrypted, so I suspect the actual limit may be slightly less?

Author: Thanks. See following revision.

> It is important to expect any number of characters in the `(string)` datatype unless directly followed by a number such as `(string36)`.
> 
> The following fields are stored together in a text datatype with a maximum length of 65,535 bytes:<br>
> First name, Last name, Address, Phone, Date of birth, Social security number

Author: Yes, we will be re-working the dev docs site soon following the structure in the brochure and partners site.

What about saying this instead? I'm trying to be as transparent as possible.
```suggestion
It is important to expect any number of characters in the `(string)` datatype unless directly followed by a number such as `(string36)`. Strings are encrypted and stored in a text datatype with a maximum length of 65,535 bytes.
```

<td markdown="1">
`address` (object)

> The [address claim](https://openid.net/specs/openid-connect-core-1_0.html#AddressClaim), containing `street_address`(string), `locality`(city, string), `region`(state, string), and `postal_code`(zip code, string5). Requires the `address` scope.

Author: Make this easier to read:
```suggestion
The [address claim](https://openid.net/specs/openid-connect-core-1_0.html#AddressClaim), containing: `street_address`(string) <br>`locality`(city, string) <br> `region`(state, string) <br> `postal_code`(zip code, string5) <br><br> Requires the `address` scope.
```

    </tr>
    <tr>
<td markdown="1">
**Social security number**<br>

> Example: 111-11-1111

Author: ```suggestion
Example:<br>`111-11-1111`
```
